Artifacts/linux-deprovision/depro.sh

```diff
@@ -2,7 +2,7 @@
 
 main () {
     { # try
-        echo 'waagent -force -deprovision+user && poweroff' | at now + 1 minute > /dev/null 2>&1
+        echo 'waagent -force -deprovision+user' | at now + 1 minute > /dev/null 2>&1
```

Comment:
Yes, I do. The `waagent` command is scheduled to run 1 minute later, using the `at` command. If we don't put a delay, the artifact will return right away and a shutdown will start. However, the `waagent` may not have fully completed its work. So, adding a delay for the shutdown will ensure that the command completes.